package.json

```
@@ -14,6 +14,7 @@
   "main": "app.js",
   "bin": "bin/dungeon-revealer",
   "scripts": {
+    "prebuild": "cross-env node scripts/prebuild.js",
```

```suggestion
    "prebuild": "node scripts/prebuild.js",
```

cross-env is not necessary if we don't provide environment variables